### PR TITLE
add buildx check if need

### PR DESCRIPTION
**Pull request by zhijianli88**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.5.md [hunk 1]
```diff
@@ -1017,7 +1017,7 @@ filename | sha256 hash
 * Correct the article in generated documents ([#32557](https://github.com/kubernetes/kubernetes/pull/32557), [@asalkeld](https://github.com/asalkeld))
 * Update PodAntiAffinity to ignore calls to subresources ([#35608](https://github.com/kubernetes/kubernetes/pull/35608), [@soltysh](https://github.com/soltysh))
 * The apiserver can now select which type of kubelet-reported address to use for apiserver->node communications, using the --kubelet-preferred-address-types flag. ([#35497](https://github.com/kubernetes/kubernetes/pull/35497), [@liggitt](https://github.com/liggitt))
-* update list of vailable resources ([#32687](https://github.com/kubernetes/kubernetes/pull/32687), [@jouve](https://github.com/jouve))
+* update list of available resources ([#32687](https://github.com/kubernetes/kubernetes/pull/32687), [@jouve](https://github.com/jouve))
```
Comment (dims) [inline CHANGELOG/CHANGELOG-1.5.md:1020]: Please remove this unrelated change

File: build/common.sh [hunk 2]
```diff
@@ -190,6 +190,10 @@ function kube::build::docker_available_on_osx() {
   fi
 }
 
+function kube::build::docker_buildx_avaliable() {
```
Comment (dims) [inline build/common.sh:193]: typo avaliable -> available

File: CHANGELOG/CHANGELOG-1.5.md (hunk 1 above)
Comment (zhijianli88) [reply CHANGELOG/CHANGELOG-1.5.md:1020]: Got it, it will submit another PR for this change

File: build/release-images.sh [hunk 3]
```diff
@@ -27,6 +27,10 @@ source "${KUBE_ROOT}/build/lib/release.sh"
 
 CMD_TARGETS="${KUBE_SERVER_IMAGE_TARGETS[*]}"
 if [[ "${KUBE_BUILD_CONFORMANCE}" =~ [yY] ]]; then
+    kube::build::docker_buildx_avaliable || {
```
Comment (dims) [inline build/release-images.sh:30]: please check spelling `docker_buildx_available`

**Comment by BenTheElder** [comment]
outstanding comments apply
